decidim-core/spec/mailers/reported_mailer_spec.rb

```diff
@@ -31,6 +31,10 @@ module Decidim
           expect(email_body(mail)).to match(moderation.participatory_space.title["en"])
         end
 
+        it "includes link to admin this resource" do
+          expect(email_body(mail)).to have_link(href: Decidim::ResourceLocatorPresenter.new(reportable).admin_url)
+        end
+
```

Comment:
We should add a new context for when the reported content is a resource without a admin url (a comment for instance), it should not have the admin url